### ISSUE TITLE
ignore end of function, or add parameter for ignore in rule newline-after-var

### ISSUE THREAD

**Issue by evilebottnawi** [eslint/eslint#3682]
```json
{
"newline-after-var": [
            2,
            "always"
        ]
}
```
```javascript
function example() {
    var anonymous = function () {
        console.log('anonymous function expression');
    };
}
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by ilyavolodin** [comment]
Could you provide details requested by the bot?

**Comment by evilebottnawi** [comment]
@ilyavolodin 
1. 1.3.1
2. 
```javascript
'use strict';

function example() {
    var anonymous = function () {
        console.log('anonymous function expression');
    };
}

example();
```
3. above
4. ESLint: Expected blank line after variable declarations. (newline-after-var)

**Comment by nzakas** [comment]
What line and column does the error message report?

**Comment by evilebottnawi** [comment]
@nzakas 
```javascript
'use strict';

function example() {
    var anonymous = function () {
        console.log('anonymous function expression');
    }; // On this line
}

example();
```

**Comment by gyandeeps** [comment]
This sounds more like an enhancement?
CC @eslint/eslint-team

**Comment by ilyavolodin** [comment]
Agree. That's an enhancement. I think it makes sense to add.

**Comment by nzakas** [comment]
What would the option be called?

**Comment by ilyavolodin** [comment]
I'm not sure this should be an option. I think it should just be an enhancement to an existing functionality.

**Comment by IanVS** [comment]
Unless there is a reason someone would want a blank line like this, and I can't imagine why anyone would, I think it would be best to avoid adding an option and just make it default behavior.

**Comment by alberto** [comment]
I'll give it a try

**Comment by ilyavolodin** [comment]
@nzakas Are you ok with enabling this without an option?

**Comment by nzakas** [comment]
Fine by me
